.github/workflows/main.yml

```diff
@@ -13,9 +13,9 @@ jobs:
       matrix:
         build: [MSRV, stable, nightly, macos, win32, win64]
         include:
-          - build: MSRV # Minimum supported Rust version
+          - build: MSRV # Minimum supported Rust version, ensure it matches the rust-version in Cargo.toml
             os: ubuntu-latest
-            rust: 1.62.1
+            rust: 1.63
```

Comment:
dup

Reply:
this is part of the #90 on which this PR was based.